### PR TITLE
Add readonly mode. (#3)

### LINKED ISSUE #3: Separate reader/writer
Thanks for your work on the project. I've been browsing the code, and I'm trying to figure out the correct way to separate readers and writers. My understanding is that clients should enable the [READONLY](https://redis.io/commands/readonly) mode. Perhaps this should be supported by the client?

Ideally I want to make sure all reads go to replicas (or slaves), whereas all writes go to masters.

### PR DESCRIPTION
**Pull request by atuk721**
Fixes #3.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -58,19 +58,66 @@ use std::thread;
...
+    pub fn password(mut self, password: String) -> Builder<T> {
+        self.password = Some(password);
+        return self;
+    }
+
+    /// Set read only mode for new Client.
+    /// Default is not read only mode.
+    /// When it is set to readonly mode, all query use replica nodes except there are no replica nodes.
+    /// If there are no replica nodes, it use master node.
+    pub fn readonly(mut self) -> Builder<T> {
```
Comment (brndnmtthws) [inline src/lib.rs]: I would include the readonly parameter in the call. For example, I may want to do:

```
builder.readonly(false);
```

File: Cargo.toml [hunk 2]
```diff
@@ -11,5 +11,5 @@ readme = "README.md"
 
 [dependencies]
 crc16 = "0.4"
-rand = "0.5"
-redis = "0.10"
+rand = "0.7"
```
Comment (brndnmtthws) [inline Cargo.toml:14]: Maybe enable dependabot for bumping dependencies?

https://github.com/marketplace/dependabot-preview

**Review by brndnmtthws — APPROVED** [review]
A couple minor comments, but over all it looks good 👍

File: src/lib.rs (hunk 1 above)
Comment (atuk721) [reply src/lib.rs]: Okay. Actually I was thinking about whether to have arguments or not.

File: Cargo.toml (hunk 2 above)
Comment (atuk721) [reply Cargo.toml:14]: Thank you for your information.
I didn't know that.
I will think about whether to do that.

**Pull request merged**